.golangci.yaml

```diff
@@ -17,3 +17,13 @@ linters-settings:
   lll:
     line-length: 228
     tab-width: 8
+  staticcheck:
+    checks:
+      - all
+      - SA1019
+issues:
+  exclude-rules:
+  - path: ".*"
+    linters:
+      - staticcheck
+    text: "SA1019"  
```

Comment:
Why ignored deprecated function? https://staticcheck.dev/docs/checks#SA1019

Reply:
Because we need to migrate a method, we now have two methods doing the same thing, but differently. Some code still uses the old and deprecated method. I added this ignore statement because the GitHub block involves code using deprecated methods.